db/schema.rb

```diff
@@ -32,6 +32,7 @@
     t.integer  "stream_id"
     t.integer  "milliseconds",                                   :default => 0
     t.float    "measured_value"
+    t.datetime "utc_time"
```

Comment:
I'd try to think of a more descriptive name. `utc_time` doesn't say what time that is

Reply:
I don't know... 
You mean it should be more specific and mention that it's for fixed sessions only? What if in the future someone wants to use it for mobile? They would have to rename. I googled and renaming columns is one of the migrations that causes down times.